v4/export/sql.go

```diff
@@ -284,6 +284,21 @@ func SetTiDBSnapshot(db *sql.DB, snapshot string) error {
 	return withStack(err)
 }
 
+func CheckTiDBWithTiKV(db *sql.DB) (bool, error) {
+	var count int
+	handleOneRow := func(rows *sql.Rows) error {
+		return rows.Scan(&count)
+	}
+	err := simpleQuery(db, "SELECT COUNT(1) as c FROM MYSQL.TiDB WHERE VARIABLE_NAME='tikv_gc_enable'", handleOneRow)
```

Comment:
perhaps better use `tikv_gc_safe_point` here, since that's what the error message talks about.

```suggestion
	err := simpleQuery(db, "SELECT COUNT(1) as c FROM MYSQL.TiDB WHERE VARIABLE_NAME='tikv_gc_safe_point'", handleOneRow)
```